yt_dlp/extractor/bilibili.py

```diff
@@ -218,6 +218,8 @@ def _real_extract(self, url):
 
         durl = traverse_obj(video_info, ('dash', 'video'))
         audios = traverse_obj(video_info, ('dash', 'audio')) or []
+        if 'flac' in video_info['dash'] and video_info['dash']['flac']['audio'] is not None:
+            audios.append(video_info['dash']['flac']['audio'])
```

Comment:
```suggestion
        flac_audio = traverse_obj(video_info, ('dash', 'flac', 'audio'))
        if flac_audio:
            audios.append(flac_audio)
```
Always avoid indexing lists/dicts/tuples directly but do it via `traverse_obj`

Reply:
Understood, working on it.

Reply:
PTAL, this has been fixed according to suggestion.